ee/vellum_ee/workflows/display/tests/test_base_workflow_display.py

```diff
@@ -522,3 +523,90 @@ class TestWorkflowDisplay(BaseWorkflowDisplay[TestWorkflow]):
     assert test_node is not None, "TestNode not found in serialized nodes"
     assert test_node["display_data"]["icon"] == "vellum:icon:cog"
     assert test_node["display_data"]["color"] == "navy"
+
+
+def test_base_workflow_display__graph_with_trigger_and_regular_node():
+    """
+    Tests that a workflow with both a trigger edge and a regular node edge serializes correctly.
+    """
+
+    # GIVEN a workflow with a trigger and regular nodes
+    class SlackMessageTrigger(IntegrationTrigger):
+        message: str
+
+        class Config:
+            provider = "COMPOSIO"
+            integration_name = "SLACK"
+            slug = "slack_message"
+
+    class TopNode(BaseNode):
+        class Outputs(BaseNode.Outputs):
+            result: str
+
+    class BottomNode(BaseNode):
+        class Outputs(BaseNode.Outputs):
+            final: str
+
+    class TestWorkflow(BaseWorkflow):
+        graph = {  # type: ignore[assignment]
```

Comment:
DevinAI remove the type ignore so I can see why this fails, I would expect this syntax to be ok